docs/reference/changelog.md

```diff
@@ -33,6 +33,7 @@ For a comprehensive list of changes, check out the [commits](https://github.com/
 - Implemented the native `fs/promise` library where possible
 - Added the functionality to output logs to the console or a file
 - In the Postgres, CrateDB, QuestDB, and TimescaleDB nodes the `Execute Query` operation returns the result from all queries executed instead of just one of the results. For more details, check out the entry in the [breaking changes](https://github.com/n8n-io/n8n/blob/master/packages/cli/BREAKING-CHANGES.md#01180) page
+- We have updated the minimum required version of Node.js to v14. For more details, check out the entry in the [breaking changes](https://github.com/n8n-io/n8n/blob/master/packages/cli/BREAKING-CHANGES.md#01180) page
```

Comment:
Same comment as above. Also, the link to breaking change for 118 does not say anything about new minimum Node.js version. Is this link incorrect or should the breaking changes entry be updated?

Reply:
@tanay1337 changed the version to 14.15. I also have a PR for the breaking change -> https://github.com/n8n-io/n8n/pull/1733